docs/design/expressions/README.md

```diff
@@ -147,17 +150,19 @@ graph BT
     expressionStatement["x;"]
 
     top --> parens & braces & unqualifiedName
+    memberAccess --> top
 
-    constType --> top
+    callAndIndexing --> memberAccess
+
+    constType --> callAndIndexing
     pointerType --> constType
     as --> pointerType
 
-    memberAccess --> top
-    pointer --> memberAccess
+    pointer --> callAndIndexing
     negation & complement & incDec --> pointer
     unary --> negation & complement
     %% Use a longer arrow here to put `not` next to `and` and `or`.
-    not -------> memberAccess
+    not -------> callAndIndexing
```

Comment:
Looking at [the indexing design](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/design/expressions/indexing.md#details), I think we were a little wrong about where this goes: function call and indexing should go in the same group as `memberAccess`, not a separate group, so that `a.b().c[i].d()` and the like work.

I think we still want the change that `constType` points to that group, though.

Reply:
Should be fixed now.